.github/ISSUE_TEMPLATE/content.md

```diff
@@ -13,17 +12,16 @@ Thank you for wanting to make amp.dev better.
 Please fill in as much of the below template as you're able.
 -->
 
-## 📖 Missing or out-of-date documentation
+## 📖 Missing or Out-of-Date Documentation
 
-**Describe the content that is missing or should be up dated.**  
-A clear and concise description of what content is missing, or needs to be updated.
-Could be a missing guide explaining a concept, a tutorial, or something that is confusing or doesn't work when you tried it.
+**Describe the content that is missing or outdated.**  
+A clear and concise description of what content is missing, or needs updating. It could be a missing guide explaining a concept, a step-by-step tutorial, or something that is confusing or doesn't work when you tried it.
 
-**In which part of the site would that content live at?**  
+**In which part of the site would that content live?**  
 Where would it go? In the reference docs? About pages? Guides and tutorials?
```

Comment:
There are too many questions here.

Reply:
@CrystalOnScript, Putting those questions in their own section was a great idea!